runbotics/runbotics-orchestrator-ui/src/src-app/views/auth/LoginPage/Login.page.tsx

```diff
@@ -88,7 +88,22 @@ const LoginPage: FC = () => {
             .catch((error) => {
                 setStatus({ success: false });
                 setSubmitting(false);
-                const status = error.status >= 400 && error.status < 500 ? '4xx' : '5xx';
+
+                if (error.status === 401 && error.data.detail !== 'Bad credentials') {
```

Comment:
i think basing on error codes (like e.g. `auth_bad_credentials`) is more reliable but that's fine